main.py

```diff
@@ -1,7 +1,36 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western",)
+    Genre.objects.create(name="Action",)
+    Genre.objects.create(name="Dramma",)
+    Actor.objects.create(first_name="George", last_name="Klooney",)
+    Actor.objects.create(first_name="Kianu", last_name="Reaves",)
```

Comment:
The first name 'Kianu' and last name 'Reaves' seem to be typos. They are later corrected to 'Keanu' and 'Reeves' in line 26, but it's better to insert the correct names initially.